low-level/sspl-ll.spec

```diff
@@ -118,6 +119,7 @@ SSPL_DIR=/opt/seagate/%{product_family}/sspl
     ln -sf $SSPL_DIR/low-level/framework/sspl_init $SSPL_DIR/bin/sspl_init
     ln -sf $SSPL_DIR/low-level/framework/sspl_reinit $SSPL_DIR/bin/sspl_reinit
     ln -sf $SSPL_DIR/low-level/framework/utils/salt_util.py $SSPL_DIR/bin/salt_util.py
+    ln -sf $SSPL_DIR/low-level/framework/utils/utility.py $SSPL_DIR/bin/utility.py
```

Comment:
utility in bin doesnt look good to me, use it from where its deployed

Reply:
Removed softlink